global-api/importer/mendoza_arg/load_mendoza_stationary_energy.sql

```diff
@@ -0,0 +1,37 @@
+-- Create a staging table
+CREATE TEMP TABLE IF NOT EXISTS mendoza_stationary_energy_staging (LIKE citywide_emissions INCLUDING ALL);
+
+-- Clear the staging table
+TRUNCATE mendoza_stationary_energy_staging;
+
+-- Load the staging table from the downloaded file
+COPY mendoza_stationary_energy_staging (year, activity_name, activity_value, activity_units, "GPC_refno", gas_name, emission_factor_value, emission_factor_units, emissions_value, emissions_units, city_name, source_name, temporal_granularity, locode, id)
+
+-- FROM '/opt/airflow/data/stationary_energy_mendoza.csv';
+FROM '/Users/maureenfonseca/Desktop/airflow-docker/scripts/stationary_energy_mendoza.csv' DELIMITER ';' CSV HEADER;
```

Comment:
Can you change this back to the path on the server? Possibly you could use relative paths here as well, so it's not dependant on the current operating system/ project location.

Reply:
got it!